.nuke/build.schema.json

```diff
@@ -126,7 +132,6 @@
               "CheckForbiddenWords",
               "Clean",
               "Compile",
-              "CopySashimiPackagesForConsolidation",
```

Comment:
I believe this is a stage that was previously removed, just the build schema wasn't committed.